### PR TITLE
created notes

### PR DESCRIPTION
**Pull request by senoodle**

### CODE REVIEW

File: spec/storeNotesSpec.js [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
 
-  
+
+  it('should store note in local storage', () => {
+    localStorage.clear();
+
+    let storeNotes = new StoreNotes();
+    storeNotes.createNote("noodle test, nooooooodles.")
+
+    let key = localStorage.key(0);
```
Comment (karimabuseer) [inline spec/storeNotesSpec.js:12]: Does this just take first value?

File: spec/testingFramework.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+const it = (label, callback) => {
+  console.log(`Test: ${label}`);
+  callback();
+}
+
+//will divide these up into helpers once we have more matchers//
+
+const expect = (a) => {
```
Comment (karimabuseer) [inline spec/testingFramework.js:15]: Nice.

File: src/displayNotes.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class DisplayNotes {
```
Comment (karimabuseer) [inline src/displayNotes.js:1]: What is this going to do? Pull notes array and filter to first 20 chars?

File: views/index.html [hunk 4]
```diff
@@ -3,15 +3,17 @@
...
-    <form id="create-note"> 
+    <form id="create-note">
       <input id="note-body" type="text" placeholder="Add your note :)"></input>
       <input type="submit" value="Create note" name="submit"></input>
     </form>
   </body>
 
-</html>
\ No newline at end of file
+</html>
```
Comment (karimabuseer) [inline views/index.html:20]: Move to line 18 and add trailing new line

**Review by karimabuseer — APPROVED** [review]

File: spec/storeNotesSpec.js (hunk 1 above)
Comment (senoodle) [reply spec/storeNotesSpec.js:12]: yesss

File: spec/testingFramework.js (hunk 2 above)
Comment (senoodle) [reply spec/testingFramework.js:15]: no u

File: src/displayNotes.js (hunk 3 above)
Comment (senoodle) [reply src/displayNotes.js:1]: exactly

**Pull request merged**